### DISCUSSION TITLE
Compiled version supporting node v22

### DISCUSSION THREAD

**Discussion by VicWilson** [julianhille/MuhammaraJS#392] [Q&A]
Hi, I'm looking to directly replace Hummus with Muhammara are there any plans to have a compiled version supporting Node v22?

**Comment by julianhille** [comment]
yes there are.

**Reply by VicWilson** [reply]
sounds good, when is it likely to land?

**Reply by julianhille** [reply]
will be there shortly.

see: https://github.com/julianhille/MuhammaraJS/pull/362

**Reply by tammannaabdullah** [reply]
Hi, I encountered an issue using Muhammara with Node 22. I then proceeded to test it in isolation with a simple file that only uses Muhammara, but it doesn't seem to work with Node 22. However, the same setup works fine on Node 20. Any suggestions on how to resolve this or insights into compatibility issues with Node 22?

**Reply by julianhille** [reply]
Would you mind opening a new issue with mote detailed information? Node v22 has landed in v5.0.0 and its building and Tests are running in the ci. Thanks.

**Comment by julianhille** [comment]
V5.0.0 has node v22 so im closing this discussion for now.
